go/kbfs/kbfsgit/runner.go

```diff
@@ -287,6 +311,22 @@ func (r *runner) isManagedByApp() bool {
 	}
 }
 
+func (r *runner) makeFS(ctx context.Context) (fs *libfs.FS, err error) {
+	// Only allow lazy creates for TLFs that aren't managed by the
+	// Keybase app.
```

Comment:
What does "managed by the Keybase app" mean?

Reply:
This was an existing comment that just moved, but it is referring to team and person private repos, the ones that are listed in the GUI and CLI.  Public and private shared repos are not properly supported by the GUI or CLI, but can still be created just by pushing to them.  (The other types of repos must be created the "official" way so they are tracked.)